### DISCUSSION TITLE
onBlur validate mode is not running

### DISCUSSION THREAD

**Discussion by julio3266** [react-hook-form/react-hook-form#8732] [General] [answered]
i'm trying validade the form using mode: onBlur but, is not running as you can see in the[ sandbox here](https://codesandbox.io/s/react-hook-form-js-forked-gbsf9?file=/src/App.tsx)

**Comment by Moshyfawn** [comment, marked as answer]
You need to pass the field's `onBlur` callback to your input's callback prop in order for RHF to register the `onBlur` event + the `ref` callback. RHF doesn't do any magic behind the scenes to track the events, it's all user managed.

```jsx
<Controller
  render={({ field }) => (
    <input onBlur={field.onBlur} ref={field.ref} />
  )}
/>
```

**Reply by julio3266** [reply]
@Moshyfawn thx, it worked for me
